modules/modes/select.js

```diff
@@ -139,42 +139,45 @@ export function modeSelect(context, selectedIDs) {
 
 
     function closeMenu() {
-        if (radialMenu) {
-            context.surface().call(radialMenu.close);
+        if (editMenu) {
+            context.surface().call(editMenu.close);
         }
     }
 
 
     function positionMenu() {
-        if (suppressMenu || !radialMenu) { return; }
+        if (suppressMenu || !editMenu) { return; }
 
         var entity = singular();
         if (entity && context.geometry(entity.id) === 'relation') {
             suppressMenu = true;
-        } else if (entity && entity.type === 'node') {
-            radialMenu.center(context.projection(entity.loc));
         } else {
             var point = context.mouse(),
-                viewport = geoExtent(context.projection.clipExtent()).polygon();
+                viewport = geoExtent(context.projection.clipExtent()).polygon(),
```

Comment:
> I also created this context.map().trimmedExtentPadding(p) for the lane work, which accepts additional padding, should we have such a thing?

Instead of adding another method, can we just have `trimmedExtent` accept an optional argument with a padding rect?